packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/styles.ts

```diff
@@ -0,0 +1,38 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    mentionFont,
+    mentionFontColor,
+    mentionDisabledFontColor
+} from '../../../../theme-provider/design-tokens';
+import { iconAtTag } from '../../../../icons/at';
+
+export const styles = css`
+    ${display('inline-block')}
+
+    :host {
+        box-sizing: border-box;
+        font: ${mentionFont};
+        white-space: normal;
+        --ni-nimble-icon-color: ${mentionFontColor};
+    }
+
+    .control {
+        color: ${mentionFontColor};
+    }
+
+    :host([disabled]) .control {
+        color: ${mentionDisabledFontColor};
+        --ni-nimble-icon-color: ${mentionDisabledFontColor};
+    }
+
+    ${iconAtTag} {
```

Comment:
While I had the icon in my example of editing vs non-editing views I'm not use if we should actually use the icon for the non-editing view or if we should have the text inserted by the template. i.e. when copy and paste as plaintext should the @ symbol copy as text.

Reply:
I believe when copy-pasting as plain text the `@` should be pasted along with the text at least for the user mention. I just quickly tried with other editors like AzDo, and GitHub which were pasted with the @ symbol. So it is better to go with the text instead of an icon for `@mention`. Later, for other types of mentions, we could use icons just like GitHub renders an icon for issues. Will that work?